arangod/RestHandler/RestAuthHandler.cpp

```diff
@@ -111,3 +112,15 @@ RestStatus RestAuthHandler::badRequest() {
                 "invalid JSON");
   return RestStatus::DONE;
 }
+
+void RestAuthHandler::shutdownExecute(bool isFinalized) noexcept {
+  TRI_DEFER(RestVocbaseBaseHandler::shutdownExecute(isFinalized));
```

Comment:
I think the TRI_DEFER is unneeded here, as the events logging is encapsulated in a try catch.
So shutdownExecute could just be called after the try..catch.